UI/css/style.css

```diff
@@ -0,0 +1,450 @@
+	padding: 1em 0;
+}
+p{
+	margin: 0;
+	padding: 1em 0;
+}
+
+.btn2{
+	display: inline-block;
+	background:#333;
+	color:#fff;
+	text-decoration:none;
+	padding: 1em 2em;
+	border: 1px solid #666;
+	margin: .5em 0;
+	width:70px;
+}
+
+.btn2:hover{
+	background: #eaeaea;
+	color: #333;
+}
+
+#showcase{
+	min-height:450px;
+	color:#fff;
+	text-align:center;
+}
+
+.white{
+	color:#5f9ea0;
+}
+
+#showcase{
+	/*position:absolute;*/
+	/*background: #fff;*/ 
+	background-color: #fff;
+	background-image: url("../images/bg2.JPG");
+	background-position: center;
+	background-repeat: no-repeat;
+	background-size: cover;
+	width: 100%;
+	height: auto;
+	z-index:-1;
+	opacity: 0.7;
+	color:black;
+}
+
+#showcase .textshow{
+		color:black;
+}
+
+#showcase.grid{
+	color:black; 
+	height:510px;
+}
+
+#showcase h1{
+	padding-top:100px;
+	padding-bottom:0;
+}
+
+#section-a{
+	background:#eaeaea;
+	color:#333;
+	padding-bottom:2em;
+}
+
+#section-b{
+	padding: 2em 1em 1em;
+	
+}
+
+#section-b ul{
+	list-style:none;
+	margin:0;
+	padding:0;
+}
+#section-b li{
+	margin-bottom:1em;
+	background:#fff;
+	color:#333;
+}
+
+.card-content{
+	padding:1.5em;
+}
+#section-d .box{
+	padding:2em;
+	color:#fff;
+}
+#section-d .box:first-child{
+	background:#2690d4;
+}
+
+.grid-body{
+	background-color:#eef1f7; 
+	width:80%; 
+	margin-right:auto; 
+	margin-left:auto; 
+	text-align:center; 
+	height:170px; 
+	margin-top:10px; 
+	border-radius:7px; 
+	line-height:30px;
+	
+}
+
+.grid-p{
+	margin-top:20px; 
+	text-align:center; 
+	width:100%; 
+	padding:0;
+}
+
+#main-footer{
+	padding:1.5em;
+	background:black;
+	color:white;
+	text-align:center;
+	height:10px;
+	margin:0px;
+}
+
+.footer1{
+	height:auto; 
+	position:fixed;
+	width:100%; 
+	padding:10px; 
+	background-color:#35475e;
+	color:white;
+	font-size:17px;
+	font-family: Arial, sans-serif;
+}
+
+.footer-table{
+	width:100%; 
+	margin:0; 
+	padding:0px; 
+	line-height:30px;
+}
+
+.footer-td{
+	width:50%; 
+	text-align:left;
+}
+
+.footer-td2{
+	width:35%; 
+	text-align:right;
+}
+
+.modal-window {
+  position: fixed;
+  background-color: rgba(200, 200, 200, 0.75);
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 999;
+  opacity: 0;
+  pointer-events: none;
+  -webkit-transition: all 0.3s;
+  -moz-transition: all 0.3s;
+  transition: all 0.3s;
+}
+
+.modal-window:target {
+  opacity: 1;
+  pointer-events: auto;
+}
+
+.modal-window>div {
+  width: 500px;
+  position: relative;
+  margin: 10% auto;
+  padding: 2rem;
+  background: #fff;
+  color: #444;
+  text-align:center;
+}
+
+.modal-window header {
+  font-weight: bold;
+}
+
+.modal-close {
+  color: #aaa;
+  line-height: 50px;
+  font-size: 80%;
+  position: absolute;
+  right: 0;
+  text-align: center;
+  top: 0;
+  width: 70px;
+  text-decoration: none;
+}
+
+.modal-close:hover {
+  color: #000;
+}
+
+.modal-window h1 {
+  font-size: 150%;
+  margin: 0 0 15px;
+}
+
+.form-modal{
+	width:60%; 
+	text-align:center; 
+	margin-right:auto; 
+	margin-left:auto;
```

Comment:
Properly align the spacing of each file, use either TABS to space or SPACEBAR and choose either 2 or 4 spaces